lib/preopt/LICENSE

```diff
@@ -217,4 +217,3 @@ conflicts with the conditions of the GPLv2, you may retroactively and
 prospectively choose to deem waived or otherwise exclude such Section(s) of
 the License, but only in their entirety and only with respect to the Combined
 Software.
-
```

Comment:
It's likely an accident of merge conflicts, but please remove this whitespace diff from the patch here.